lib/datadog/tracing/trace_operation.rb

```diff
@@ -160,6 +160,13 @@ def service
         @service || (root_span && root_span.service)
       end
 
+      # Returns tracer tags that will be propagated if this span's context
+      # is exported through {.to_digest}.
+      # @return [Hash] key value pairs of distributed tags
+      def distributed_tags
+        meta.select { |name, _| name.start_with?(Metadata::Ext::Distributed::TAGS_PREFIX) }
```

Comment:
This is likely a good candidate function to make private. I don't think there's any need for this in the public scope.